app/views/events/index.html.erb

```diff
@@ -1,31 +1,98 @@
-<p id="notice"><%= notice %></p>
-
-<h1>Listing Events</h1>
 
-<table>
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Content</th>
-      <th>User</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
+<nav class="navbar">
```

Comment:
Let's move nav bar functionality to a partial and put it in `applicartion.html.erb`